frontend/src/lib/components/ActivityLog/activityLogLogic.tsx

```diff
@@ -98,6 +71,19 @@ export const activityLogLogic = kea<activityLogLogicType<CountedPaginatedRespons
             },
         ],
     }),
+    listeners: ({ actions }) => ({ setPage: actions.fetchNextPage }),
+    urlToAction: ({ values, actions }) => {
+        const onPageChange = (searchParams: Record<string, any>): void => {
+            const pageInURL = searchParams['page']
+            if (pageInURL && pageInURL !== values.page) {
+                actions.setPage(pageInURL)
+            }
+        }
+        return {
+            '/person/*': ({}, searchParams) => onPageChange(searchParams),
+            [urls.featureFlags()]: ({}, searchParams) => onPageChange(searchParams),
+        }
+    },
     events: ({ actions }) => ({
```

Comment:
Is there any way to make this aware of the `props`? Currently if we have two activity log logics mounted at the same time, both will change their page no matter if you go on a feature flag or person activity page. 

Also, I suggest making a new url/route, `urls.featureFlagsActivity()`. Currently this runs also on the feature flags index page. What if we want to start paginating that as well?